definitions/marts/looker_studio_marts/ecf/ls_ecf_assurance_bad_shape_1_out_of_sequence.sqlx

```diff
@@ -0,0 +1,163 @@
+config {
+    type: "table",

```

Comment:
Suggest adding a Dataform tag to all these marts so you can run them all together quickly in the pipeline if needs be

Reply:
And also a BigQuery label to the tables produced

Reply:
See https://cloud.google.com/dataform/docs/configure-additional-actions#add-tag and https://cloud.google.com/dataform/docs/create-tables#labels